src/components/location/dto/create-location.dto.ts

```diff
@@ -40,6 +47,39 @@ export abstract class CreateCountry {
   readonly regionId: string;
 }
 
+@InputType()
+export abstract class CreatePrivateLocation {
```

Comment:
This comment still stands https://github.com/SeedCompany/cord-api-v3/pull/928#pullrequestreview-466050489
We'll have a Location interface, with PublicLocation and PrivateLocation implementations. Then mutations will be for a single "location" object.

Reply:
Hmm.... I am just confused about this comment. If possible, can you rewrite the `Location` in the DTOs and I will refactor service/resolver following that type.